common/Jackdmp.cpp

```diff
@@ -202,6 +202,7 @@ static void usage(FILE* file, jackctl_server_t *server, bool full = true)
             "               [ --port-max OR -p maximum-number-of-ports]\n"
             "               [ --slave-backend OR -X slave-backend-name ]\n"
             "               [ --internal-client OR -I internal-client-name ]\n"
+            "               [ -C configuration-file ]\n"
```

Comment:
All the other options have long and short versions. Maybe the -C option also should have a long version.